src/appMain/hmi_capabilities.json

```diff
@@ -349,7 +349,7 @@
 },
     "TTS":
 	{
-		"capabilities":"TEXT",
+		"capabilities":["TEXT"],
```

Comment:
@istoilovgithub I think you need to change method load_capabilities_from_file from hmi_capabilities_impl.cc.